tests/test_trainer.py

```diff
@@ -1093,17 +1094,13 @@ def test_training_iterable_dataset(self):
         self.assertIsInstance(loader.sampler, torch.utils.data.dataloader._InfiniteConstantSampler)
 
     def test_training_finite_iterable_dataset(self):
-        num_gpus = max(1, get_gpu_count())
-        if num_gpus > 2:
-            return
-
         config = RegressionModelConfig()
         model = RegressionPreTrainedModel(config)
 
         batch_size = 1
         num_samples = 10
 
-        available_steps = num_samples // (batch_size * num_gpus)
+        available_steps = num_samples // batch_size
```

Comment:
This was all added for multi-processing, but in the test we use DataParallel, not DistributedDataParallel